### DISCUSSION TITLE
front end

### DISCUSSION THREAD

**Discussion by zchubez** [community/community#47529] [New to GitHub] [answered]
### Select Topic Area

Product Feedback

### Body

will developer front end need?

**Comment by al1103** [comment, marked as answer]
html ,css, js

**Comment by unknowVariable** [comment]
Front-end developers typically need to have a solid understanding of HTML, CSS, and JavaScript, as well as familiarity with various front-end frameworks such as React, Angular, and Vue.js. They also need to have a good eye for design and user experience, as well as the ability to collaborate effectively with designers, product managers, and other developers.

In addition to technical skills, front-end developers may also benefit from having strong communication and problem-solving skills, as they often work closely with other team members to solve complex development challenges and ensure that the final product meets both technical and business requirements. Additionally, staying up-to-date with new technologies and trends in front-end development can also be important for remaining competitive in the field.

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @zchubez 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
